api/src/main/java/org/pmiops/workbench/profile/ProfileService.java

```diff
@@ -144,7 +148,8 @@ public Profile getProfile(DbUser userLite) {
         freeTierDollarQuota,
         accessTierShortNames,
         userTierEligibilities,
-        accessModules);
+        accessModules,
+        newUserSatisfactionSurveyService);
```

Comment:
alt: could instead calculate the result value above this call (like freeTierUsage), and pass in the boolean here instead of the service.  That way, you wouldn't need to add an `@AfterMapper` method.

Reply:
https://github.com/all-of-us/workbench/pull/7135/commits/d5e6798206898ff7923fc557d12f8e67c41387d5 removes the `@AfterMapping` test, so this is no longer tested.

I'm personally still on the fence about tests that are 100% mocking/stubbing verification. I see we don't have a general test for `getProfile`, so I didn't add one.

Let me know if I should add one.

Reply:
no I don't think it's necessary here